reflex/.templates/web/utils/state.js

```diff
@@ -240,7 +240,13 @@ export const applyEvent = async (event, socket) => {
  */
 export const applyRestEvent = async (event, socket) => {
   let eventSent = false;
-  if (event.handler == "uploadFiles") {
+  if (event.handler === "uploadFiles") {
+
+    if (event.payload.files === undefined || event.payload.files.length === 0){
+      // Submit the event over the websocket to trigger the event handler.
+      return await applyEvent(Event(event.name, event.payload.files), socket)
```

Comment:
the payload is supposed to be a object, not an array

Reply:
you mean `return await applyEvent(Event(event.name, event.payload), socket)`

Reply:
the problem is that `event.payload` contains keys that do not map to an arbitrary event handler on the backend.

for example, maybe it works if the backend event handler has a `files` arg; but if that arg is named `fools`, then we get an error on the backend about unknown keyword argument.